lib/src/debug_page.dart

```diff
@@ -3,33 +3,33 @@ import 'package:flutter_ebook_app/src/common/common.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 
 class DebugPage extends ConsumerWidget {
-  const DebugPage({super.key});
+  const DebugPage({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final isDarkMode = ref.watch(currentAppThemeNotifierProvider).value ==
-        CurrentAppTheme.dark;
+    final isDarkMode = context.read(currentAppThemeNotifierProvider).value == CurrentAppTheme.dark;
```

Comment:
the theme needs to be **watched** not **read**

Reply:
Alright